ophyd/flyers.py

```diff
@@ -259,11 +265,26 @@ def _get_stream_name(self, attr):
         obj = getattr(self, attr)
         return self.stream_names.get(attr, obj.name)
 
+    def _describe_with_dtype(self, attr, *, dtype='array'):
```

Comment:
not enough to not merge it, but it is a nice way to enforce readable function calls in cases where there is not a good default.